diff -urN grace-5.1.2/configure grace-5.1.2-PDFlib/configure
--- grace-5.1.2/configure	Sun Sep 10 14:14:00 2000
+++ grace-5.1.2-PDFlib/configure	Thu Dec  7 01:13:26 2000
@@ -7738,8 +7738,8 @@
     pdf_library=-lpdf
   fi
 
-  echo $ac_n "checking "for PDFlib \>= 3.02"""... $ac_c" 1>&6
-echo "configure:7743: checking "for PDFlib \>= 3.02"" >&5
+  echo $ac_n "checking "for PDFlib \>= 3.0"""... $ac_c" 1>&6
+echo "configure:7743: checking "for PDFlib \>= 3.0"" >&5
 if eval "test \"`echo '$''{'acx_cv_pdflib'+set}'`\" = set"; then
   echo $ac_n "(cached) $ac_c" 1>&6
 else
@@ -7771,7 +7771,7 @@
         vlibn = 100*PDF_get_majorversion() + PDF_get_minorversion();
         vincn = 100*PDFLIB_MAJORVERSION + PDFLIB_MINORVERSION;
         vinc = PDFLIB_VERSIONSTRING;
-        if (strcmp(vinc, "3.02") < 0) {
+        if (strcmp(vinc, "3.0") < 0) {
           exit(1);
         }
         if (vincn != vlibn) {
