--- grace-5.1.12/src/t1fonts.c.orig	Thu Apr  3 20:57:12 2003
+++ grace-5.1.12/src/t1fonts.c	Thu Apr  3 21:41:00 2003
@@ -85,7 +85,7 @@
         return (RETURN_FAILURE);
     }
     
-    nfonts = T1_Get_no_fonts();
+    nfonts = T1_GetNoFonts();
     if (nfonts < 1) {
         return (RETURN_FAILURE);
     }
--- grace-5.1.12/src/fontwin.c.orig	Wed Aug 21 22:19:51 2002
+++ grace-5.1.12/src/fontwin.c	Thu Apr  3 21:51:41 2003
@@ -320,7 +320,7 @@
     Widget font_table = (Widget) data;
     
     FontID = value;
-    switch (CheckForFontID(FontID)) {
+    switch (T1_CheckForFontID(FontID)) {
     case 0:
         T1_LoadFont(FontID);
         break;
