--- grace-5.1.22/src/rstdrv.c~	2004-07-03 22:47:46.000000000 +0200
+++ grace-5.1.22/src/rstdrv.c	2012-02-12 22:16:50.451780508 +0100
@@ -54,6 +54,7 @@
 
 #ifdef HAVE_LIBPNG
 #  include <png.h>
+#  include <zlib.h>
 #endif
 
 #ifndef NONE_GUI
@@ -886,7 +886,7 @@
         return;
     }
 
-    if (setjmp(png_ptr->jmpbuf)) {
+    if (setjmp(png_jmpbuf(png_ptr))) {
         png_destroy_write_struct(&png_ptr, &info_ptr);
         return;
     }
